tests/virt/node/log_verbosity/test_log_virt_launcher.py

```
@@ -87,8 +87,8 @@ def source_pod_log_verbosity_test(admin_client, vm_for_migration_progress_test):
 
 
 @pytest.fixture()
-def migrated_vm_with_policy(migration_policy_with_bandwidth, vm_for_migration_progress_test):
-    migrate_vm_and_verify(vm=vm_for_migration_progress_test, wait_for_migration_success=False)
+def migrated_vm_with_policy(admin_client, migration_policy_with_bandwidth, vm_for_migration_progress_test):
```

Ruff (0.15.14)

[warning] Unused function argument: `migration_policy_with_bandwidth`

(ARG001)